cms/static/sass/elements/_forms.scss

```diff
@@ -172,8 +172,97 @@ form {
   }
 }
 
+// +Forms - License selector UI
+// ====================
+.license-img {
+  padding: 4px;
+}
+
+ul.license-types {
+  text-align: middle;
+  vertical-align: middle;
+  display: inline-block;
+
+  li {
```

Comment:
There is no class applied to these elements. Should there be? Why shouldn't we use element selectors?